Pods/PathKit/Sources/PathKit.swift

```diff
@@ -673,6 +673,6 @@ internal func +(lhs: String, rhs: String) -> Path {
 
 extension Array {
   var fullSlice: ArraySlice<Element> {
-    return self[0..<self.endIndex]
+    return self[self.indices]
```

Comment:
Oh, sorry for that, should have been more careful with search & replace. Fixed now in a288a41e900df5b16e60b2e6a279f85d885a17d2.